depthai_nodes/ml/messages/creators/misc.py

```diff
@@ -47,3 +50,53 @@ def create_age_gender_message(age: float, gender_prob: List[float]) -> AgeGender
     age_gender_message.gender = gender
 
     return age_gender_message
+
+
+def create_multi_classification_message(
```

Comment:
Should we move it to `creators/classification.py` since it is still classification-related msg?

Reply:
Will move it in next PR where I also update AgeGender to CompositMessage as @jkbmrz  mentioned.